docs/getting-started/README.md

```diff
@@ -1,31 +1,31 @@
 # 🚀 Getting started
 
-Welcome! 👋
+👋 Welcome! You've just taken your first step on the path to mastering Frontity.
 
-Frontity is a **React framework** to create **WordPress themes**. It allows you to easily build a React frontend for a headless WordPress site, which serves its data via the WP REST API. Frontity framework is open source and free to use.
+Frontity is a **React-based framework** that enables you to easily build a frontend for a headless (or decoupled) WordPress site. Your WordPress site serves its data via the REST-API. Frontity framework is open source and free to use.
 
-## Get started
+## Installation requirements
 
-You can get started right now, it will be really easy! You may check:
+To get started with Frontity you will need:
 
-* 📘 [**Quick start guide**](quick-start-guide.md)**:** here you will find how to create a website with Frontity, step by step.
-* 👨‍👩‍👧‍👦 [**Community forum**](https://community.frontity.org/)**:** the perfect place to ask any questions and share your feedback.
+* **A WordPress installation** - This can be hosted locally, on a web-server, or you can also use a site hosted on wordpress.com. However, when you spin up your first Frontity project you will initially be connected to our test site so you won't be without content when you first try Frontity.
 
-Frontity also welcomes contributions. There are many ways to support the project and get involved. Plus, you can get **free swag**! Please refer to the guide below to learn more about it:
+* **Node installed in your computer** - If you don't already have it you can get **Node** from [the official site](https://nodejs.org/). Alternatively, you can install Node using **nvm** \([node version manager](https://github.com/creationix/nvm)\). Any of these methods will also install **npm** and **npx** along with Node. You will use these to run Frontity commands during the set-up and development of your project.
```

Comment:
I would add here how you can check if (or when) you have node installed
Something like...
```
⬢  ~  node -v
v10.15.3
⬢  ~  npm -v
6.13.1
```

Reply:
Also, remove the `nvm` explanation part as just using the latest version is enough for most of the cases.

Reply:
Removed nvm explanation and added check for node/npm